analysis/config/config.go

```diff
@@ -286,12 +242,9 @@ type Options struct {
 	// in the folder the binary is called.
 	ReportsDir string `xml:"reports-dir,attr" yaml:"reports-dir" json:"reports-dir"`
 
-	// PathSensitiveFuncs is a list of regexes indicating which functions should be path-sensitive.
-	// This allows the analysis to scale yet still maintain a degree of precision where it matters.
-	PathSensitiveFuncs []string `xml:"field-sensitive-funcs" yaml:"field-sensitive-funcs" json:"field-sensitive-funcs"`
-
-	// pathSensitiveFuncsRegexes is a list of compiled regexes corresponding to PathSensitiveFuncs
-	pathSensitiveFuncsRegexes []*regexp.Regexp
+	// ProjectRoot specifies the root directory of the project. All other file names specified in the config file are
+	// relative to the root. If not specified, the root is assumed to be the directory of the config file.
+	ProjectRoot string `xml:"project-root,attr" yaml:"project-root" json:"project-root"`
 
 	// SkipInterprocedural can be set to true to skip the interprocedural (inter-procedural analysis) step
 	SkipInterprocedural bool `xml:"skip-interprocedural,attr" yaml:"skip-interprocedural" json:"skip-interprocedural"`
```

Comment:
We should probably change this to `UnsafeSkipInterprocedural` because it can lead to unsound results

Reply:
It really doesn't produce any results, it was used just as a sanity check that summary building doesn't fail. I wonder if we should just remove it.

Reply:
Removed it.